shell/platform/darwin/ios/framework/Source/FlutterPlatformPluginTest.mm

```diff
@@ -17,10 +17,29 @@ @interface FlutterPlatformPluginTest : XCTestCase
 
 @interface FlutterPlatformPlugin ()
 - (BOOL)isLiveTextInputAvailable;
+- (void)showLookUpView:(NSString*)term;
 @end
 
 @implementation FlutterPlatformPluginTest
 
+- (void)testLookUpCallInitiated {
+  FlutterEngine* engine = [[[FlutterEngine alloc] initWithName:@"test" project:nil] autorelease];
+  std::unique_ptr<fml::WeakPtrFactory<FlutterEngine>> _weakFactory =
+      std::make_unique<fml::WeakPtrFactory<FlutterEngine>>(engine);
+  XCTestExpectation* invokeExpectation = [self expectationWithDescription:@"isLookUpInvoked"];
+  FlutterPlatformPlugin* plugin =
+      [[[FlutterPlatformPlugin alloc] initWithEngine:_weakFactory->GetWeakPtr()] autorelease];
+  FlutterPlatformPlugin* mockPlugin = OCMPartialMock(plugin);
+  FlutterMethodCall* methodCall = [FlutterMethodCall methodCallWithMethodName:@"LookUp.initiate"
+                                                                    arguments:@"Test"];
+  FlutterResult result = ^(id result) {
+    OCMVerify([mockPlugin showLookUpView:@"Test"]);
```

Comment:
The `showLookUpView` function itself is not tested here. Can you mock the vc and verify `presentViewController` is called?